packages/types/src/files-ref.ts

```diff
@@ -0,0 +1,3 @@
+export function folderPathToFilesRef(pathToFolder: string): string {
+  return encodeURIComponent(pathToFolder.replaceAll('/', '-').replaceAll('\\', '-').replaceAll('_', '')) + '.json';
```

Comment:
We could also use a regex if we want to instead of 2 `replaceAll()` calls. But there might be a reason you went with the string approach instead?

```suggestion
  return encodeURIComponent(pathToFolder.replaceAll(/[\/\\]+/g, '-').replaceAll('_', '')) + '.json';
```

Reply:
No good reason except that it was easier to write! Let's use the regex instead :+1: